api/v1alpha2/mondooauditconfig_types.go

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2022 Mondoo, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// MondooAuditConfigSpec defines the desired state of MondooAuditConfig
+type MondooAuditConfigData struct {
```

Comment:
Calling it `Data` has always struck me as a weird thing. I'm used to seeing `.*Spec` here in CRD definitions.
```suggestion
type MondooAuditConfigSpec struct {
```